### DISCUSSION TITLE
Accidentally bought a 3 pin 3.5 mm jack microphone and a 4 pin 3.5 mm jack USB sound card.

### DISCUSSION THREAD

**Discussion by Oripy** [mcguirepr89/BirdNET-Pi#1101] [Q&A] [answered]
When I connected my system with components I just bought it wasn't working (nothing was recorded, the spectrograph was empty). After a long investigation I figured my issue is that I bought a 4 pins USB sound card (https://www.amazon.fr/dp/B08Y8CZB2S?psc=1&ref=ppx_yo2ov_dt_b_product_details) and a 3 pins microphone (FEL Communications Clippy EM272 Mono Microphone).

Is there an easy way to get this working without buying new hardware? Should I just buy a new USB Sound card? If so what do you recommend?

**Comment by blogdemoi** [comment]
You can buy a TRS to TRRS adapter maybe ? : https://www.amazon.fr/s?k=trs+to+trrs&i=electronics&__mk_fr_FR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=2SW80QGL10GGP&sprefix=trs+to+trrs%2Celectronics%2C243&ref=nb_sb_noss_1

**Reply by Oripy** [reply, marked as answer]
I bought an other USB soundcard with the proper connector.
Thank you for the input.
